README.md

```diff
@@ -1,62 +1,86 @@
+[![](https://user-images.githubusercontent.com/1248214/57789522-600fcc00-7739-11e9-86d9-73d7032f40fc.png)
+](https://kilt.io)
 
-![](https://user-images.githubusercontent.com/1248214/57789522-600fcc00-7739-11e9-86d9-73d7032f40fc.png)
+Data sovereignty and interoperability
 
-# KILT Testnet (Mash-net) SDK
+# Mash-Net SDK
 
-The KILT SDK is a collection of classes and methods that application developers can utilize to interact with the KILT Network. The SDK is provided in Typescript.
+The open-source KILT's SDK testnet is written in TypeScript and enables developers to build, play and design permissionless blockchain apps and businesses. [KILT](https://kilt.io) network provides a system for self-sovereign data and interoperability. The SDK provides collection of classes and methods developers can utilize to interact with the KILT Network.
```

Comment:
Wording suggestion: use "You" instead of "developers"

Reply:
As we are directing it at the developer's then yes.